marketplace_docs/templates/Fabric/interactive_script.sh

```diff
@@ -18,11 +18,42 @@ EOM
 read -r _
 
 while  [ -z "${email}" ] || [ -z "${hostname}" ]; do
-read -r -p "Email address for support and error emails (e.g. ops@example.com): " email
-read -r -p "Hostname (e.g. chat.example.com): " hostname
+    read -r -p "Email address for support and error emails (e.g. ops@example.com): " email
+    read -r -p "Hostname (e.g. chat.example.com): " hostname
+    echo ""
+done
+
+SERVICES_FLAGS=""
 echo ""
+echo "Would you like to automatically register your server for push notifications?"
+echo "For details on why a centralized push notification service is necessary, see:"
+echo "    <https://zulip.readthedocs.io/en/latest/production/mobile-push-notifications.html#why-a-push-notification-service-is-necessary>"
+read -r -p "Answer [Y/n]: " push_prompt
+case "${push_prompt,,}" in
+    "" | y | yes)
+        SERVICES_FLAGS="--push-notifications"
 
-done
+        echo ""
+        echo "Do you agree for your server to submit aggregate usage statistics to the push notification service?"
+        echo "For details, see:"
+        echo "    <https://zulip.readthedocs.io/en/latest/production/mobile-push-notifications.html#uploading-usage-statistics>"
+        read -r -p "Answer [Y/n]: " stats_prompt
+        case "${stats_prompt,,}" in
+            n | no)
+                echo "Upload of aggregate usage statistics will not be enabled."
```

Comment:
```suggestion
                echo "Aggregate usage statistics will not be uploaded."
```

Is this clear enough? In the current version, "Upload" as a noun is a bit hard to parse.

Reply:
And a parallel tweak below if this is fine.